### ISSUE TITLE
Fix lstm_stock_prediction dataset location

### ISSUE THREAD

**Issue by rcurtin** [mlpack/examples#130]
If you try to `make` and run the `lstm_stock_prediction` example after downloading the dataset as per the README, it doesn't work:

```
$ ./lstm_stock_prediction 
Reading data ...
[FATAL] Cannot open file 'Google2016-2019.csv'. 

terminate called after throwing an instance of 'std::runtime_error'
  what():  fatal error; see Log::Fatal output
Aborted
```

It should be a pretty easy fix; I think the filename just needs to be fixed.

**Comment by rcurtin** [comment]
(See also #128.)

**Comment by heisenbuug** [comment]
`download_data_set.py1 is creating a folder called data and placing all the datasets in that folder...this was a basic file path issue...
Line 135 is causing the issue...
currently: `  const string dataFile = "Google2016-2019.csv";`
change: `const string dataFile = "../data/Google2016-2019.csv";`

**Comment by rcurtin** [comment]
Thanks @heisenbuug!  This is fixed in #137.
